config/initializers/hyrax.rb

```diff
@@ -139,15 +139,17 @@
   # config.binaries_directory = "tmp/binaries"
 
   # If browse-everything has been configured, load the configs.  Otherwise, set to nil.
-  begin
-    if defined? BrowseEverything
-      config.browse_everything = BrowseEverything.config
-    else
-      Rails.logger.warn "BrowseEverything is not installed"
-    end
-  rescue Errno::ENOENT
-    config.browse_everything = nil
-  end
+  # TODO: Re-enable this when work on BE has been prioritized
```

Comment:
I guess that's not appropriate now that browse_everything is a hard dependency of Hyrax: https://github.com/samvera-labs/hyrax/blob/68527862df498aa9f407b909d4f41d781ba146dc/app/jobs/import_url_job.rb#L50